### PR TITLE
chore: Update isWalletASmartWallet to isWalletACoinbaseSmartWallet

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Update `isWalletASmartWallet` naming to be `isWalletACoinbaseSmartWallet` so we explicitly state what the function is doing. 
**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,13 +67,20 @@
...
   "publishConfig": {
     "access": "public"
   },
-  "files": ["esm/**/*", "lib/**/*", "src/", "src/**/*"],
+  "files": [
+    "esm/**/*",
+    "lib/**/*",
+    "src/",
+    "src/**/*"
+  ],
```
Comment (cpcramer) [inline package.json:83]: Auto formatting from new CI lint

**Comment by lukasrosario** [comment]
if we want this util would maybe go with `isAddressACoinbaseSmartWallet` instead, wallet is just the client, what we're really checking is the address

**Comment by lukasrosario** [comment]
hmm actually this accepts a user op. so maybe something like `isCoinbaseSmartWalletUserOp` ?

ultimately i still think we should avoid utilities like these if possible and lean on capabilities. so i'd challenge this by asking if there is a specific use case for this that cant be answered by querying capabilities

**Review by Zizzamia — APPROVED** [review]
Approving and merging this for now. But yeah we need to re-think the capabilities approach, and how to make it simpler for folks to use them.

**Pull request merged**